full/Dockerfile

```
@@ -245,4 +245,7 @@ RUN for i in $(ls /var/lib/apt/dazzle-marks/*.gpg); do apt-key add "$i"; done
 # copy tests to enable the self-test of this image
 COPY tests /var/lib/dazzle/tests
 
+# share env see https://github.com/gitpod-io/workspace-images/issues/472
+RUN echo "PATH="${PATH}"" | sudo tee /etc/environment 
```

Whoops, please remove the trailing whitespace here (sorry, our validation checks are a bit unhelpful 🙄):

```suggestion
RUN echo "PATH="${PATH}"" | sudo tee /etc/environment
```

---

fixed.